frontend/src/utils/dateUtil.ts

```diff
@@ -0,0 +1,13 @@
+// Convert "2024-02-21" to "February 21, 2024"
+export function formatDate(dateStr: string | null) {
```

Comment:
Nice! Can we do something like this for currency? My solution for the award amount formatting feels kinda hacky.

Reply:
Yep - I propose  a separate ticket/pr though since this one getting big.